readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -0,0 +1,79 @@
+# Read the Docs configuration file
+
+# The version of the spec to be use
+version: enum('2')
+
+# Formats of the documentation to be built
+# Default: ['htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia']
+formats: any(list(enum('htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia')), enum('all'), required=False)
+
+# The path to the requirements file from the root of the project
+requirements_file: str(required=False)
+
+# Configuration for Conda support
+conda: include('conda', required=False)
+
+# Configuration for the documentation build process
+build: include('build', required=False)
+
+# Configuration of the Python executable to be used
+python: include('python', required=False)
+
+# Configuration for sphinx
+sphinx: include('sphinx', required=False)
+
+# Submodules configuration
+submodules: include('submodules', required=False)
+
+# Redirects for the current version to be built
+redirects: include('redirects', required=False)
+
+---
+
+conda:
+  # The path to the Conda environment file from the root of the project
+  file: str()
+
+build:
+  # The build docker image to be used
+  # Default: '2.0'
+  image: enum('1.0', '2.0', 'latest')
+
+python:
+  # The Python version
+  # Default: '2.7'
+  version: enum('2', '2.7', '3', '3.3', '3.4', '3.5', '3.6', required=False)
+
+  # Install the project using python setup.py install or pip
+  install: enum('pip', 'setup.py', required=False)
+
+  # Extra requirements sections to install in addition to the package dependencies
+  # Default: []
+  extra_requirements: list(str(), required=False)
+
+  # Give the virtual environment access to the global site-packages dir
+  # Default: false
+  system_packages: bool(required=False)
+
+sphinx:  
+  # The path to the conf.py file
+  # Default: rtd will try to find it
+  file: str(required=False)
+
+submodules:
+  # List of submodules to be included
+  # Default: []
+  include: any(list(str()), enum('all'), required=False)
+
+  # List of submodules to be ignored
+  # Default: []
+  exclude: any(list(str()), enum('all'), required=False)
+  
+  # Do a recursive clone?
+  # Default: false
+  recursive: bool(required=False)
+
+redirects:
+  # Key/value list, represent redirects of type page
+  # from url -> to url
+  page: map(str(), str())
```

Comment:
For redirects, this is the only type that makes sense here. I didn't add the html_to_dir type, I think that's more like a global setting.

Reply:
Maybe we can just drop the `page` key and use only `redirects`

Reply:
Yea, we likely need a type as well (since we have a few types). So perhaps it's a list of dicts, with the dict being `{type, from, to}` or similar.